### PR TITLE
[PDR-912[ Update/add genomic tables for PDR.

### PR DESCRIPTION
**Pull request by robabram**
## Description of changes/additions
* Update existing PDR models with removed and new genomics table fields 
* Add 'genomic_informing_loop' table to resources.
* Added initial support for sending Pub/Sub events for new RDR to PDR data pipeline. (non-prod for now).

## Tests
- [x] unit tests

### CODE REVIEW

**Review by dtharpeuno — APPROVED** [review]

File: requirements_base.txt [hunk 1]
```diff
@@ -73,7 +73,7 @@ parameterized==0.7.4      # via -r requirements.in
 pbr==5.4.5                # via jira
 pdfminer.six==20201018
 # pip-tools==5.1.2          # via -r requirements.in
-protobuf==3.15.0          # via -r requirements.in, google-api-core, google-cloud-bigquery, googleapis-common-protos
+protobuf==3.20.1          # via -r requirements.in, google-api-core, google-cloud-bigquery, googleapis-common-protos
```
Comment (pegbertsch) [inline requirements_base.txt:76]: What triggered this requirements update?

File: rdr_service/resource/schemas/genomics.py [hunk 2]
```diff
@@ -177,6 +216,7 @@ class GenomicManifestFileSchema(Schema):
     rdr_processing_complete_date = fields.DateTime()
     ignore_flag = fields.Int16()
     file_name = fields.String(validate=validate.Length(max=255))
+    manifest_type_id_str = fields.String(validate=validate.Length(max=64))
```
Comment (pegbertsch) [inline rdr_service/resource/schemas/genomics.py]: Maybe this isn't the convention for the genomics tables, but there is an enum class for these manifest types (`rdr_service/genomic_enums.py GenomicManifestTypes` .  Do we want to have the `manifest_type` / `manifest_type_id` paired fields like we do for other PDR schemas?

File: rdr_service/resource/schemas/genomics.py [hunk 3]
```diff
@@ -93,12 +93,11 @@ class GenomicSetMemberSchema(Schema):
     cvl_w4f_manifest_job_run_id = fields.Int32()
     genomic_workflow_state = fields.EnumString(enum=GenomicWorkflowState)
     genomic_workflow_state_id = fields.EnumInteger(enum=GenomicWorkflowState)
-
-    genomic_workflow_state_history = fields.JSON()
+    # genomic_workflow_state_history = fields.JSON()
```
Comment (pegbertsch) [inline rdr_service/resource/schemas/genomics.py]: What triggered the commenting out of the fields in this table?

File: requirements_base.txt (hunk 1 above)
Comment (robabram) [reply requirements_base.txt:76]: If you jump to the github home page for the RDR project, it is displaying a library version vulnerability alert. https://github.com/all-of-us/raw-data-repository.
This takes care of that alert.

File: rdr_service/resource/schemas/genomics.py (hunk 2 above)
Comment (robabram) [reply rdr_service/resource/schemas/genomics.py]: Good catch, I didn't pay attention to this field.  The RDR devs have added an enum string field for the 'manifest_type_id' field, but we already handled having both the id and str fields previously.  I have removed the new field as it is not needed, in my next commit.

File: rdr_service/resource/schemas/genomics.py (hunk 3 above)
Comment (robabram) [reply rdr_service/resource/schemas/genomics.py]: I commented out those fields when I found they had been removed from the RDR table.  I have removed them in my next PR.

File: rdr_service/resource/schemas/genomics.py (hunk 2 above)
Comment (robabram) [reply rdr_service/resource/schemas/genomics.py]: Turns out I was adding a few unneeded enum string fields.  I should have looked harder at those new fields.  I'm glad you caught this.

**Review by pegbertsch — APPROVED** [review]
Newline/codacy behavior is just FYI.  The other Codacy issues are based on var/function naming that we don't want to change so there won't be a clean Codacy run regardless.

**Pull request merged**